examples/plot_2D_simulation_example.py

```diff
@@ -267,8 +274,11 @@ def plot(maps, titles):
 )
 
 # clustered desparsified lasso (CluDL)
-beta_hat, pval, pval_corr, one_minus_pval, one_minus_pval_corr = clustered_inference(
-    X_init, y, ward, n_clusters
+ward_, beta_hat, theta_hat, omega_diag = clustered_inference(
+    X_init, y, ward, n_clusters, scaler_sampling=StandardScaler()
+)
+beta_hat, pval, pval_corr, one_minus_pval, one_minus_pval_corr = (
```

Comment:
Getting back to our discussion: rather have a single function that does the job and a class that allows to introspect model components.

Reply:
I agree but before changing the API, I prefer to have a homogeneous API for all the methods. 
For the next API, see the issue #201.